web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -0,0 +1,200 @@
+          };
+        }
+        if (selectedType === "MarkdownText") {
+          values.typeProperty = {
+            markdownText: { defaultValue: values.defaultValue, maxLength: values.maxLength },
+          };
+        }
+        if (selectedType === "Asset") {
+          values.typeProperty = {
+            asset: { defaultValue: values.defaultValue.uid },
+          };
+        }
+        if (selectedType === "Select") {
+          values.typeProperty = {
+            select: { defaultValue: values.defaultValue, values: values.values },
+          };
+        }
+        if (selectedType === "Integer") {
+          values.typeProperty = {
+            integer: { defaultValue: +values.defaultValue, min: +values.min, max: +values.max },
+          };
+        }
+        if (selectedType === "URL") {
+          values.typeProperty = {
+            url: { defaultValue: values.defaultValue },
+          };
+        }
+
+        await onSubmit?.({
+          ...values,
+          fieldId: selectedField?.id,
+        });
+        form.resetFields();
+        onClose?.(true);
+      })
+      .catch(info => {
+        console.log("Validate Failed:", info);
+      });
+  }, [form, onClose, onSubmit, selectedType, selectedField?.id]);
+
+  const handleClose = useCallback(() => {
+    form.resetFields();
+    onClose?.(true);
+  }, [onClose, form]);
+
+  return (
+    <Modal
+      title={
+        selectedType ? (
+          <FieldThumbnail>
+            <StyledIcon
+              icon={fieldTypes[selectedType].icon}
+              color={fieldTypes[selectedType].color}
+            />{" "}
+            <h3>Update {fieldTypes[selectedType].title}</h3>
+          </FieldThumbnail>
+        ) : null
+      }
+      visible={open}
+      onCancel={handleClose}
+      onOk={handleSubmit}>
+      <Form form={form} layout="vertical" initialValues={initialValues}>
+        <Tabs defaultActiveKey="settings">
+          <TabPane tab="Setting" key="setting">
+            <Form.Item
+              name="title"
+              label="Display name"
+              rules={[{ required: true, message: "Please input the display name of field!" }]}>
+              <Input />
+            </Form.Item>
+            <Form.Item
+              name="key"
+              label="Field Key"
+              rules={[{ required: true, message: "Please input the key of field!" }]}>
+              <Input />
+            </Form.Item>
+            <Form.Item requiredMark="optional" name="description" label="Description">
+              <TextArea rows={3} showCount maxLength={100} />
+            </Form.Item>
+            <Form.Item name="multiValue" extra="Stores a list of values instead of a single value">
+              <Checkbox>Support multiple values</Checkbox>
+            </Form.Item>
+          </TabPane>
+          <TabPane tab="Validation" key="validation">
+            <FieldValidationProps selectedType={selectedType}></FieldValidationProps>
```

Comment:
I've mentioned this before. Please don't do <dadsf> </dadsf> if there is no children being passed. Just <dadsd />

Reply:
Please update everywhere that its like this